viziphant/gpfa.py

```diff
@@ -83,7 +83,7 @@ def plot_dimension_vs_time(returned_data,
         orthonmalization, these dimensions reflect mixtures of timescales.
     plot_single_trajectories : bool
         If True, single trial trajectories are plotted.
-    n_trials_to_plot : int
+    num_trials_to_plot : int
```

Comment:
I think we should stick to Elephant convention of `n_` and not `num_`.
This is minor anyway